Modelica/Blocks/Logical.mo

```diff
@@ -21,7 +21,10 @@ the output is <strong>false</strong>.
   end And;
 
   block MultiAnd "Logical and of Boolean vector elements"
-    extends Modelica.Blocks.Interfaces.PartialBooleanMISO;
```

Comment:
Why are you switching from connector with automatic sizing  to vector input without that feature?
To me that just seems like worse usability.

If you want another name for the input that's fine - but it can still have automatic sizing.

(Oh, and how do you comment on an entire commit - not on specific lines?)

Reply:
It seems there are two possibilities how to implement this and I agree with @HansOlsson that it is indeed confusing to combine them both. Esp. if the classes' naming suggests the same handling.

Reply:
To me this suggests that we could improve the GUI for connectorSizing (both in tools and in the non-normative part of the specification).
Note that the second proposal written in textual form should still work with connectorSizing.

Reply:
No it doesn't (I tried it).